src/SDDK/wf_ortho.cpp

```diff
@@ -46,30 +46,31 @@ orthogonalize(::spla::Context& spla_ctx__, memory_t mem__, linalg_t la__, int is
     auto spins = (ispn__ == 2) ? std::vector<int>({0, 1}) : std::vector<int>({ispn__});
 
     int K{0};
+    using precision_type = typename real_type<T>::type;
     if (sddk_pp) {
         K = wfs__[0]->gkvec().num_gvec() + wfs__[0]->num_mt_coeffs();
-        if (std::is_same<T, double>::value) {
+        if (std::is_same<T, precision_type>::value) {
             K *= 2;
         }
     }
 
     auto sddk_debug_ptr = utils::get_env<int>("SDDK_DEBUG");
     int sddk_debug      = (sddk_debug_ptr) ? (*sddk_debug_ptr) : 0;
 
-    double ngop{0};
-    if (std::is_same<T, double>::value) {
+    precision_type ngop{8e-9};
```

Comment:
ngop is the number of floating-point operations (int giga-units) for the matrix-matrix multiplications. For real type it is 2*m*n*k, for complex types it is 8*m*n*k. Basically, it is there to report FLOPS. It can always be a `double` data type.